docs/source/install.md

```diff
@@ -252,64 +270,92 @@ until [ -z "$(docker service ls --filter label=com.docker.stack.namespace=secret
 echo "Removing existing secrets"
+
 docker secret rm FLOWMACHINE_FLOWDB_PASSWORD || true
 docker secret rm FLOWMACHINE_FLOWDB_USER || true
 docker secret rm FLOWAPI_FLOWDB_PASSWORD || true
 docker secret rm FLOWAPI_FLOWDB_USER || true
-docker secret rm POSTGRES_PASSWORD || true
+docker secret rm FLOWDB_POSTGRES_PASSWORD || true
 docker secret rm cert-flowkit.pem || true
-docker secret rm JWT_SECRET_KEY || true
 docker secret rm REDIS_PASSWORD || true
 docker secret rm FLOWAPI_IDENTIFIER || true
+docker secret rm FLOWAUTH_ADMIN_USERNAME || true
+docker secret rm FLOWAUTH_ADMIN_PASSWORD || true
+docker secret rm FLOWAUTH_DB_PASSWORD || true
+docker secret rm PRIVATE_JWT_SIGNING_KEY || true
+docker secret rm PUBLIC_JWT_SIGNING_KEY || true
+
+# Add new secrets
 
 echo "Adding secrets"
+# Flowauth
+openssl genrsa -out tokens-private-key.key 4096
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWAUTH_DB_PASSWORD -
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWAUTH_ADMIN_PASSWORD -
+echo "admin" | docker secret create FLOWAUTH_ADMIN_USERNAME -
+pip install cryptography && python -c "from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())" | docker secret create FLOWAUTH_FERNET_KEY -
+
+
+# Flowmachine
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWMACHINE_FLOWDB_PASSWORD -
 echo "flowmachine" | docker secret create FLOWMACHINE_FLOWDB_USER -
-openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' \
-    | docker secret create FLOWMACHINE_FLOWDB_PASSWORD -
 echo "flowapi" | docker secret create FLOWAPI_FLOWDB_USER -
-openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWAPI_FLOWDB_PASSWORD -
-openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create POSTGRES_PASSWORD -
 openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create REDIS_PASSWORD -
+
+# FlowDB
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWAPI_FLOWDB_PASSWORD -
+openssl rand -base64 16 | tr -cd '0-9-a-z-A-Z' | docker secret create FLOWDB_POSTGRES_PASSWORD -
+
+# FlowAPI
+openssl rsa -pubout -in  tokens-private-key.key -out tokens-public-key.pub
+docker secret create PRIVATE_JWT_SIGNING_KEY tokens-private-key.key
+docker secret create PUBLIC_JWT_SIGNING_KEY tokens-public-key.pub
 echo "flowapi_server" | docker secret create FLOWAPI_IDENTIFIER -
 openssl req -newkey rsa:4096 -days 3650 -nodes -x509 -subj "/CN=flow.api" \
     -extensions SAN \
     -config <( \
-        cat $(   ( [[ -e /System/Library/OpenSSL/openssl.cnf ]] \
-                   && echo "/System/Library/OpenSSL/openssl.cnf" ) \
-              || ( [[ -e /etc/ssl/openssl.cnf ]] \
-                   && echo "/etc/ssl/openssl.cnf" ) \
-              || ( [[ -e /etc/pki/tls/openssl.cnf ]] \
-                   && echo "/etc/pki/tls/openssl.cnf" ) ) \
+        cat $(   ( [[ -e /System/Library/OpenSSL/openssl.cnf ]] && echo "/System/Library/OpenSSL/openssl.cnf" ) \
```

Comment:
I agree it would be nicer to have these all on a single line. The reason I broke this up in a previous PR is because the docs page is fairly narrow, so the complete line isn't visible if it's too long, which I personally find irritating because you need to keep scrolling right and left if you want to follow the script. Would you be happy to keep these lines split so that they are fully readable in the docs without horizontal scrolling?